inflation/main.py

```diff
@@ -0,0 +1,22 @@
+from fastapi import FastAPI
+import logging
+from inflation.dataset.crawl import Crawler
+
+
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+app = FastAPI()
+crawler = Crawler()
+
+
+def fetch_inflation_data():
+    url = "https://docs.google.com/spreadsheets/d/1mZSqW_X_KuQQdGhH7oAn3-MATyfePLlYYmOgE-mB_9Q/edit#gid=0"
+    records = crawler.parse_excel_to_link_dataset(url)
+    crawler.crawl(records, "/inflation-resources/data/")
+
+
+@app.get("/crawl", tags=["crawl"])
+async def fetch_data():
+    fetch_inflation_data()
```

Comment:
I think it is good but, one thing worries me is that it can take for a while to fetch this data. we may need to create thread or separate process to handle this part and return the message immediately. it is a bit advanced, we can talk about it later. for now it is good.